src/main/java/com/exasol/dbbuilder/dialects/mysql/MySQLIdentifier.java

```diff
@@ -0,0 +1,34 @@
+package com.exasol.dbbuilder.dialects.mysql;
+
+import com.exasol.db.Identifier;
+
+/**
+ * MySQL-specific identifier.
+ */
+public class MySQLIdentifier implements Identifier {
```

Comment:
Please create ticket to add validation according to
https://dev.mysql.com/doc/refman/8.0/en/identifiers.html

Reply:
done 
https://github.com/exasol/test-db-builder-java/issues/19